bammetrics/src/main/scala/nl/lumc/sasc/biopet/pipelines/bammetrics/BammetricsReport.scala

```diff
@@ -85,10 +86,10 @@ object BammetricsReport extends ReportBuilder {
 
     ReportPage(
       if (targets.isEmpty) List()
-      else List("Targets" -> ReportPage(
+      else List("Targets" -> Future(ReportPage(
```

Comment:
This should be `Future.succcessful` instead of just `Future` to avoid spawning a new `Future`.